src/ProxyClient/Varnish.php

```diff
@@ -60,14 +60,17 @@ class Varnish extends AbstractProxyClient implements BanInterface, PurgeInterfac
 
     /**
      * {@inheritdoc}
+     *
+     * @param string $tagsHeader Header for tags, defaults to X-Cache-Tags
      */
     public function __construct(
         array $servers,
         $baseUri = null,
-        HttpAdapter $httpAdapter = null,
+        HttpAsyncClient $httpClient = null,
+        MessageFactory $messageFactory = null,
         $tagsHeader = 'X-Cache-Tags'
```

Comment:
should we go with an options parameter for this, like in the Symfony proxy client?